runtime/wasm/src/module/mod.rs

```diff
@@ -998,7 +998,12 @@ where
             }
             ETHEREUM_CALL_FUNC_INDEX => {
                 let _section = stopwatch.start_section("host_export_ethereum_call");
-                self.ethereum_call(args.nth_checked(0)?)
+                let arg = if self.ctx.host_exports.api_version >= Version::new(0, 0, 4) {
+                    self.asc_get::<_, AscUnresolvedContractCall_0_0_4>(args.nth_checked(0)?)
```

Comment:
It isn't clear to me how the call function indices differ per APIVersion. Does the `_0_0_4` suffix simply mean that struct was created for version 0.0.4?

Reply:
I believe you introduced this `_x_y_z` version notation yourself. (I could be wrong about this 😉.)

`_0_0_4` means introduced in `apiVersion: 0.0.4`.

Reply:
Indeed I did. 😊